openshift/e2e-tests-openshift.sh

```diff
@@ -5,7 +5,7 @@ source $(dirname $0)/release/resolve.sh
 
 set -x
 
-readonly SERVING_VERSION=v0.7.1
+readonly SERVING_VERSION=v0.6.0
```

Comment:
Was this supposed to be 0.8.0?

Reply:
no - that's to make this build work.

soon will be TP1 (0.7.1)